package.json

```
@@ -354,6 +354,178 @@
 						}
 					}
 				}
+			},
+			{
+				"type": "reactnativedirect",
+				"label": "React Native Direct",
+				"program": "./src/debugger/direct/reactNativeDirectDebugEntryPoint.js",
+				"runtime": "node",
+				"enableBreakpointsFor": {
+					"languageIds": [
+						"javascript",
+						"typescript",
+						"javascriptreact",
+						"typescriptreact"
+					]
+				},
+				"configurationSnippets": [
+					{
+						"label": "React Native (Hermes): Debug Android",
+						"description": "%reactNative.snippets.debugAndroid.description%",
```

```suggestion
						"description": "%reactNative.snippets.debugAndroidHermes.description%",
```

Please, also update package.nls.json.